src/pages/workspace/workflows/WorkspaceWorkflowsPage.tsx

```diff
@@ -51,7 +51,7 @@ function WorkspaceWorkflowsPage({policy, betas, route}: WorkspaceWorkflowsPagePr
 
     const policyApproverEmail = policy?.approver;
     const policyApproverName = useMemo(() => PersonalDetailsUtils.getPersonalDetailByEmail(policyApproverEmail ?? '')?.displayName ?? policyApproverEmail, [policyApproverEmail]);
-    const containerStyle = useMemo(() => [styles.ph8, styles.mhn8, styles.ml11, styles.pv3, styles.pr0, styles.pl4, styles.mr0, styles.widthAuto, styles.mt4], [styles]);
+    const containerStyle = useMemo(() => [styles.ph8, styles.mhn5, styles.ml0, styles.pv3, styles.pr0, styles.pl2, styles.mr0, styles.widthAuto, styles.mt4, styles.pb0], [styles]);
```

Comment:
You can merge styles like `ml0` and `mr0` into `mh0`, same for the paddding:

This should be sufficient (try it before commiting 😅):
```ts
[styles.pt3, styles.pr0, styles.pl2, styles.pb0, styles.mh0, styles.mt4, styles.widthAuto]
```

Reply:
Container style is no longer needed, so it solve itself haha :)